swap_meet/decor.py

```diff
@@ -1,2 +1,29 @@
-class Decor:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Decor(Item):
+    def __init__(self, id=None, condition=0, width=0,length=0):
```

Comment:
Nitpick: missing whitespace
```suggestion
    def __init__(self, id=None, condition=0, width=0, length=0):
```